src/AgValue.cpp

```diff
@@ -763,6 +795,20 @@ JSONVar Measurements::buildPMS(AirGradient &ag, int ch, bool allCh, bool withTem
     pms["channels"]["2"]["pm003Count"] = ag.round2(_pm_03_pc[1].update.avg);
   }
 
+  /// PM0.5 particle count
+  if (utils::isValidPm03Count(_pm_05_pc[0].update.avg) &&
+      utils::isValidPm03Count(_pm_05_pc[1].update.avg)) {
+    float avg = (_pm_05_pc[0].update.avg + _pm_05_pc[1].update.avg) / 2.0f;
+    pms["pm005Count"] = ag.round2(avg);
+    pms["channels"]["1"]["pm005Count"] = ag.round2(_pm_05_pc[0].update.avg);
```

Comment:
005 vs 05, I think we should align this to avoid confusion

Reply:
I just follow the existing formatting. Is it better if we use `pms05Count` and `pms50Count` or `pms5Count`? If yes, shouldn't we need to update other field too then?

Reply:
Changes
- `pm25Count` to `pm02Count`
- `pm05Count` to `pm50Count`

Detail on `local-server.md`